test/01-constrain.test.ts

```diff
@@ -0,0 +1,78 @@
+import * as supertest from 'supertest';
+import { expect } from 'chai';
+const fixturePath = __dirname + '/fixtures/01-constrain/config';
+import { testInit, testDeInit, testLocalServer } from './lib/testInit';
+
+describe('01 basic constrain tests', function () {
+	let pineServer: any;
+	before(async () => {
+		pineServer = await testInit(fixturePath, true);
+	});
+
+	describe('Basic', () => {
+		it('check /ping route is OK', async () => {
+			const res = await supertest(testLocalServer).get('/ping').expect(200);
+			expect(res.text).to.equal('OK');
+		});
+	});
+
+	describe('university vocabular', () => {
+		it('check /university/student is served by pinejs', async () => {
+			const res = await supertest(testLocalServer)
+				.get('/university/student')
+				.expect(200);
+			expect(res.body)
+				.to.be.an('object')
+				.has.ownProperty('d')
+				.to.be.an('array');
+		});
+
+		it('create a student', async () => {
+			await supertest(testLocalServer)
+				.post('/university/student')
+				.send({
+					matrix_number: 1,
+					name: 'John',
+					lastname: 'Doe',
+					birthday: new Date(),
+					semester_credits: 10,
+				})
+				.expect(201);
+		});
+
+		it('should fail to create a student with same matrix number ', async () => {
+			await supertest(testLocalServer)
+				.post('/university/student')
+				.send({
+					matrix_number: 1,
+					name: 'John',
+					lastname: 'Doe',
+					birthday: new Date(),
+					semester_credits: 10,
+				})
+				.expect(409);
+		});
+
+		it('should fail to create a student with too few semester credits ', async () => {
+			const res = await supertest(testLocalServer)
+				.post('/university/student')
+				.send({
+					matrix_number: 2,
+					name: 'Jenny',
+					lastname: 'Dea',
+					birthday: new Date(),
+					semester_credits: 2,
+				})
+				.expect(400);
+			expect(res.body)
+				.to.be.an('string')
+				.eq(
```

Comment:
If you're going for the English style it makes sense to do it fully:
```suggestion
				.that.equals(
```

Reply:
thx, changed.